### PR TITLE
MVP - 3.0.0 alpha release!

### PR DESCRIPTION
**Pull request by aviramha**
Thank you for contributing to mirrord!

Please make sure you added a CHANGELOG entry & squashed your PR to one commit.

### CODE REVIEW

**Review by infiniteregrets — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+
```
Comment (eyalb181) [inline .vscode/launch.json:7]: ?

File: Cargo.toml [hunk 2]
```diff
@@ -13,7 +13,7 @@ members = [
 
 # latest commits on rustls suppress certificate verification
 [workspace.package]
-version = "2.13.0"
+version = "3.0.0-alpha"
```
Comment (eyalb181) [inline Cargo.toml:16]: No channel?

File: vscode-ext/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	"displayName": "mirrord",
 	"description": "Mirror live traffic from your Kubernetes cluster to your local debugged process.",
 	"publisher": "MetalBear",
-	"version": "2.13.0",
+	"version": "3.0.0",
```
[the remark this answers is not in the document]
Comment (aviramha) [reply vscode-ext/package.json:6]: can't on vscode marketplace :(

[the remark this answers is not in the document]
Comment (infiniteregrets) [reply vscode-ext/package.json:6]: i think --pre-release solves it?

File: Cargo.toml (hunk 2 above)
Comment (aviramha) [reply Cargo.toml:16]: no such thing in rust toml

File: vscode-ext/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (aviramha) [reply vscode-ext/package.json:6]: "solves" (best we can do in vs code market place :()

**Pull request merged**